fe/src/main/java/org/apache/doris/catalog/Type.java

```diff
@@ -822,29 +828,38 @@ public Integer getNumPrecRadix() {
         compatibilityMatrix[FLOAT.ordinal()][DOUBLE.ordinal()] = PrimitiveType.DOUBLE;
         compatibilityMatrix[FLOAT.ordinal()][DATE.ordinal()] = PrimitiveType.INVALID_TYPE;
         compatibilityMatrix[FLOAT.ordinal()][DATETIME.ordinal()] = PrimitiveType.INVALID_TYPE;
+        compatibilityMatrix[FLOAT.ordinal()][TIME.ordinal()] = PrimitiveType.TIME;
         compatibilityMatrix[FLOAT.ordinal()][VARCHAR.ordinal()] = PrimitiveType.INVALID_TYPE;
         compatibilityMatrix[FLOAT.ordinal()][CHAR.ordinal()] = PrimitiveType.INVALID_TYPE;
         compatibilityMatrix[FLOAT.ordinal()][HLL.ordinal()] = PrimitiveType.INVALID_TYPE;
 
         compatibilityMatrix[DOUBLE.ordinal()][DATE.ordinal()] = PrimitiveType.INVALID_TYPE;
         compatibilityMatrix[DOUBLE.ordinal()][DATETIME.ordinal()] =
                 PrimitiveType.INVALID_TYPE;
+        compatibilityMatrix[DOUBLE.ordinal()][TIME.ordinal()] = PrimitiveType.TIME;
         compatibilityMatrix[DOUBLE.ordinal()][VARCHAR.ordinal()] = PrimitiveType.INVALID_TYPE;
         compatibilityMatrix[DOUBLE.ordinal()][CHAR.ordinal()] = PrimitiveType.INVALID_TYPE;
         compatibilityMatrix[DOUBLE.ordinal()][HLL.ordinal()] = PrimitiveType.INVALID_TYPE;
 
         compatibilityMatrix[DATE.ordinal()][DATETIME.ordinal()] = PrimitiveType.DATETIME;
+        compatibilityMatrix[DATE.ordinal()][TIME.ordinal()] = PrimitiveType.INVALID_TYPE;
         compatibilityMatrix[DATE.ordinal()][VARCHAR.ordinal()] = PrimitiveType.INVALID_TYPE;
         compatibilityMatrix[DATE.ordinal()][CHAR.ordinal()] = PrimitiveType.INVALID_TYPE;
         compatibilityMatrix[DATE.ordinal()][HLL.ordinal()] = PrimitiveType.INVALID_TYPE;
 
+        compatibilityMatrix[DATETIME.ordinal()][TIME.ordinal()] = 
+                PrimitiveType.INVALID_TYPE;
         compatibilityMatrix[DATETIME.ordinal()][VARCHAR.ordinal()] =
                 PrimitiveType.INVALID_TYPE;
         compatibilityMatrix[DATETIME.ordinal()][CHAR.ordinal()] =
                 PrimitiveType.INVALID_TYPE;
         compatibilityMatrix[DATETIME.ordinal()][HLL.ordinal()] =
                 PrimitiveType.INVALID_TYPE;
 
+        compatibilityMatrix[TIME.ordinal()][VARCHAR.ordinal()] = PrimitiveType.VARCHAR;
+        compatibilityMatrix[TIME.ordinal()][CHAR.ordinal()] = PrimitiveType.VARCHAR;
+        compatibilityMatrix[TIME.ordinal()][HLL.ordinal()] = PrimitiveType.INVALID_TYPE;
```

Comment:
TIME.ordinal is bigger than HLL CHAR and VARCHAR, So I think there is some problem here.